src/core/Akka.Cluster.Tests/ClusterDeployerSpec.cs

```diff
@@ -85,6 +94,26 @@ public void RemoteDeployer_must_be_able_to_parse_akka_actor_deployment_with_spec
             deployment.Dispatcher.ShouldBe("mydispatcher");
         }
 
+        [Fact]
+        public void BugFix2266RemoteDeployer_must_be_able_to_parse_broadcast_group_cluster_router_with_default_nr_of_routees_routees()
```

Comment:
Reproduction spec which verifies that we weren't parsing the correct number of routees originally: https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka.Cluster/Configuration/Cluster.conf#L209